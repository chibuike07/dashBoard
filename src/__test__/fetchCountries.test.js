import Axios from "axios";

test("should includes countries", async () => {
  await Axios.get(
    `https://randomuser.me/api/?page=${1}&results=${10}&gender=${"all users"}&noinfo`,
    {
      "Content-Type": "application/json",
    }
  ).then((res) => {
    const countries = res.data.results.map(({ location: { country } }) => {
      return country;
    });
    expect(countries).toEqual(expect.arrayContaining([]));

    // expect(countries).toMatchSnapshot();
  });
});
