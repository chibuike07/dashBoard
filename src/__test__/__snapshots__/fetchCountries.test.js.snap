// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should includes countries 1`] = `
Array [
  "Brazil",
  "Netherlands",
  "Australia",
  "France",
  "Turkey",
  "Australia",
  "New Zealand",
  "Ireland",
  "United Kingdom",
  "Netherlands",
]
`;
